.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+jobs:
+  build-deploy:
+    docker:
+      - image: circleci/python:3.7-node-browsers
+    steps:
+      - checkout
+      - restore_cache:
+          key: deps-{{ .Branch }}-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install aws-cli
+          command: sudo pip3 install awscli
+      - run:
+          name: Install node dependencies
+          command: yarn install
+      - save_cache:
+          key: deps-{{ .Branch }}-{{ checksum "yarn.lock" }}
+          paths:
+            - "node_modules"
+      - run: 
+          name: Deploy CFT
+          command: aws cloudformation deploy --template .circleci/cloudformation.yml --stack-name smash-overlays --region us-west-1 --no-fail-on-empty-changeset
+          no_output_timeout: 60m
```

Comment:
60m seems long?

Reply:
Sometimes necessary if we update a cert or CDN. I had to leave it running in a screen session on Pichu for the first deployment